templates/notebook_instance/sdk_update_pre_build_request.go.tpl

```diff
@@ -0,0 +1,16 @@
+if err = rm.requeueUntilCanModify(ctx, latest); err != nil {
+	return latest, err
+}
+latestStatus := latest.ko.Status.NotebookInstanceStatus
+// The Notebook Instance is stopped and the StoppedByController status is
+// set to UpdatePending.
+if latestStatus != nil &&
+ *latestStatus == svcsdk.NotebookInstanceStatusInService {
+	if err := rm.stopNotebookInstance(latest); err != nil {
+		return latest, err
+	} else {
+		//TODO: Take this out if the runtime supports updating annotations if an error is returned and use annotations for this.
+		latest.ko.Status.StoppedByControllerMETA = aws.String("UpdatePending")
+	  return latest, requeueWaitWhileStopping
```

Comment:
This else block seems to be malformed (or is just the indentation on github ?)

Reply:
It's just a github thing, looks fine on vscode